src/wp-includes/class-wp-scripts.php

```diff
@@ -390,8 +430,20 @@ public function do_item( $handle, $group = false ) {
 			return true;
 		}
 
+		if ( '' !== $strategy ) {
+			$strategy = ' ' . $strategy;
+			if ( ! empty( $after_non_standalone_handle ) ) {
+				$strategy .= sprintf( " onload='wpLoadAfterScripts(\"%s\")'", esc_attr( $handle ) );
```

Comment:
This should be more robust to guard against malicious script handles:

```suggestion
				$strategy .= sprintf( " onload='wpLoadAfterScripts(%s)'", esc_attr( wp_json_encode( $handle ) ) );
```